docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,591 @@
+[role="xpack"]
+[[azure-module]]
+=== Azure Module  [Experimental]
+experimental[]
+
+The https://azure.microsoft.com/en-us/overview/what-is-azure/[Microsoft Azure] module in Logstash helps you easily integrate your Azure
+activity logs and SQL diagnostic logs with the Elastic Stack. 
+
+NOTE: The Logstash Azure module is an
+https://www.elastic.co/products/x-pack[{xpack}] feature under the Basic License
+and is therefore free to use. Please contact
+mailto:monitor-azure@elastic.co[monitor-azure@elastic.co] for questions or more
+information.
+
+The Azure module uses the
+{logstash-ref}/plugins-inputs-azure_event_hubs.html[Logstash Azure Event Hubs
+plugin] to consume data from Azure Event Hubs. The module taps directly into the
+Azure dashboard, parses and indexes events into Elasticsearch, and installs a
+suite of {kib} dashboards to help you start exploring your data immediately.   
+
+[[azure-dashboards]]
+==== Dashboards
+
+These {kib} dashboards are available and ready for you to use. 
+
+* *Overview*. Top-level view into your Azure operations, including info about users, resource groups, service health, access, activities, and alerts. 
```

Comment:
Should we group these into two logical sections?
- "Infrastructure Activity Monitoring" - Overview, User Activity, and Alerts
- "SQL DB Monitoring" - SQL DB Overview, SQL DB Database View, SQL DB Queries